install

```diff
@@ -13,69 +19,125 @@ usage() {
   exit 1
 }
 
-INSTALLED_VIA_HOMEBREW=false
-
+# Reads all options passed in and sets environment variables or shows usage/help
 while getopts ":pub" opt; do
   case $opt in
     p ) SKIP_SETTING_PATH=1;;
-    u ) DO_UNINSTALL=1;;
```

Comment:
The help text and the cask file still contain the `-u` option flag.

Reply:
Oops! Thanks for catching that